### ISSUE TITLE
Suggestion: allow saving bakery settings in a scriptableobject

### ISSUE THREAD

**Issue by pedropla** [guycalledfrank/bakery-issues#126]
I have quite a few scenes that share similar settings so it would be great if there was a way of saving them into a settings file in a scriptableobject or something. Perhaps with an option to either auto sync or load/save depending on the scene. 

I know this isn't a bug or a very urgent issue, but when dealing with many scenes that share similar bakery settings it would help prevent mistakes when forgetting to update a field somewhere. If you don't feel it's worth it feel free to just close this issue. :)

**Comment by guycalledfrank** [comment]
Happy new year!

I was thinking of doing that, but check if you can already use existing solutions, like:

a) Bakery->Utilities->Save settings as default / Load default Settings. If you have mainly one set of common settings for the project, that'll work. "Default" is stored in ftGlobalStorage.asset.

b) Create an object with Bakery Lightmapped Prefab. Click "Save current render settings to prefab". Create the prefab from this object. Drop it to another scene, click "Load render settings from prefab" (object can be deleted then).

**Comment by pedropla** [comment]
That would work somewhat. However, having them as scriptable objects would allow me to better optimize my workflow. That way I could have configurations for scenes that need higher resolution (smaller, perhaps more indoor scenes), large open world lower resolution scenes, and for those with other requirements. Then when I run a process to bake all the scenes I wouldn't have to first open each scene and save changes and hope it saved correctly to the bakery object, etc.

Basically I'm trying to reduce the need to open scenes that are part of a baking "batch" if I need to make a baking change to all of them. That would reduce the risk of missing some and then having to rebake after what is already a pretty long process.

**Comment by danieldownes** [comment]
Options as part of the batch bake:
"Batch bake with current settings"
or
"Batch bake with each scenes settings"
